### ISSUE TITLE
Allow linking to layer metadata

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/HTML-Map-Element-UseCases-Requirements#6]
https://github.com/Maps4HTML/MapML/issues/36

**Comment by AmeliaBR** [comment]
To clarify, this is about including metadata _within_ the map layer data file?

**Comment by prushforth** [comment]
I believe it is about linking to "metadata", in the sense of a descriptive document that follows a metadata standard for geospatial data, for example ISO 19115.

**Comment by AmeliaBR** [comment]
Yes, my question is, where is the link coming from?  The web page, or the map data file (e.g., a MapML file)?

**Comment by prushforth** [comment]
Yes, the link would be in a MapML document, the content that would be retrieved would be some sort of descriptive metadata about the map data visible in the MapML document. Not sure if rel=via is the right or even only link rel that would fit that use case. I imagine there is an established practice with HTML; we should lean on that.
